src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -942,7 +942,7 @@ public void getBlob(
     Set<String> workerSet;
     Set<String> locationSet;
     try {
-      workerSet = backplane.getWorkers();
+      workerSet = backplane.getStorageWorkers();
```

Comment:
IME the definition of [`No workers found.`](https://github.com/bazelbuild/bazel-buildfarm/pull/1328/files#diff-7d364025210bb06a44a529f14efcdf9a0bbbd8e715af51786c4f0ca24310fe52R1097) could use some tweaking. Generally if you don't have a storage worker ( which prior happened to mean exec + storage ) it will give an `UNAVAILABLE` message.

Do you think it now makes more sense to rename this `No storage workers` found? If the CAS goes down it's a problematic situation. The reality of "No execution workers available" is probably undesirable to assert here unless we want that to mean "Hey - you don't have any execute workers"; I wonder if one day `GetCapabilities` could be more dynamic

Reply:
A buildfarm could theoretically transition from exec <-> cas only depending what's up-and-running - or all your exec nodes popped

Reply:
Also the situation of 0 storage workers makes it probably hard to assert: say you have some external CAS